lib/danger_plugin.rb

```diff
@@ -1,6 +1,3 @@
 # frozen_string_literal: true
 
-require 'dangermattic/plugins/manifest_pr_checker'
-require 'dangermattic/plugins/milestone_checker'
-require 'dangermattic/plugins/pr_size_checker'
-require 'dangermattic/plugins/view_changes_need_screenshots'
+Dir[File.join(__dir__, 'dangermattic/plugins', '*.rb')].each { |file| require file }
```

Comment:
You could also make use of `require_relative` instead:
```suggestion
Dir['dangermattic/plugins/*.rb'].each do |file|
  require_relative file
end
```

Reply:
This is something I tried in the early versions but it didn't work -- just tried here again and the tests fail with `uninitialized constant Danger::<plugin>`.